### PR TITLE
fix(input): size variant for input/dropdown

### PR DESCRIPTION
**Pull request by yashasvi13**
### JIRA Ticket

Jira:  [AN-1715](https://hello-haptik.atlassian.net/browse/AN-1715)

--

### Changelog

Size variant small for input and integrated for dropdown

### Test link
https://60c9aea10a6a260007e0d401--gifted-wing-304c8e.netlify.app/?path=/story/components-inputs-text--documentation

### CODE REVIEW

File: src/components/Inputs/Dropdown/Dropdown.js [hunk 1]
```diff
@@ -294,6 +296,7 @@ const Dropdown = ({
               focused={focused}
               isReversed={isReversed}
               icon={icon}
+              size={size}
```
Comment (sidMulay) [inline src/components/Inputs/Dropdown/Dropdown.js:299]: If we're only adding a small within the size variant, would it work if we only name the prop as a flag? 
eg. prop can be a boolean like just `small`. If dev adds `small` to the dropdown, it'll render smaller one.

File: src/components/Inputs/Text/Text.js [hunk 2]
```diff
@@ -75,6 +76,7 @@ const Text = ({
         themed={themed}
         label={label}
         style={style}
+        size={size}
```
Comment (sidMulay) [inline src/components/Inputs/Text/Text.js:79]: Same as above.

File: src/components/Inputs/_common/ThemedTextInput.js [hunk 3]
```diff
@@ -112,6 +121,7 @@ ThemedTextInput.defaultProps = {
   mode: "default",
   variant: "default",
   fontSize: "2",
+  size: "",
```
Comment (sidMulay) [inline src/components/Inputs/_common/ThemedTextInput.js:124]: Why was this added here? I can see that some padding has been added to the textarea

File: src/Theme/textInput.js [hunk 4]
```diff
@@ -47,3 +47,9 @@ export const textInputVariants = {
     borderTopLeftRadius: 0,
   },
 };
+
+export const textInputSize = {
+  small: {
+    padding: `${space.sm}px ${space.lg - space.sm}px`,
```
Comment (sidMulay) [inline src/Theme/textInput.js:53]: Are we only reducing padding and not the font size?

**Review by sidMulay — CHANGES_REQUESTED** [review]

File: src/Theme/textInput.js (hunk 4 above)
Comment (yashasvi13) [reply src/Theme/textInput.js:53]: Yeah fontsize remains the same

File: src/components/Inputs/Dropdown/Dropdown.js (hunk 1 above)
Comment (sidMulay) [reply src/components/Inputs/Dropdown/Dropdown.js:299]: As discussed, this suggestion is parked for now since size can possibly expand to large

**Review by sidMulay — APPROVED** [review]

**Pull request merged**